contracts/governance/InterchainGovernance.sol

```diff
@@ -183,20 +184,6 @@ contract InterchainGovernance is AxelarExecutable, TimeLock, Caller, IInterchain
         return keccak256(abi.encode(target, callData, nativeValue));
     }
 
-    /**
-     * @notice Overrides internal function of AxelarExecutable, will always revert
-     * as this governance module does not support execute with token.
-     */
-    function _executeWithToken(
-        string calldata, /* sourceChain */
-        string calldata, /* sourceAddress */
-        bytes calldata, /* payload */
-        string calldata, /* tokenSymbol */
-        uint256 /* amount */
-    ) internal pure override {
-        revert TokenNotSupported();
```

Comment:
example of why it becomes cleaner for devs

Reply:
Agreed.